tests/language/typing.liq

```diff
@@ -152,10 +152,10 @@ def f() =
   end
   ignore(sum_eq)
 
-  (noise():source(audio=pcm,video=canvas,midi=none))
```

Comment:
Is `midi=none` still supported? I think we should for best backward compat...

Reply:
In this specific case I believe that it is implicit:
```
(s:source)
```
Is an open type `source('a)`

But:
```
(s:source(audio=pcm, video=canvas)
```
is a closed type: `source(audio=pcm, video=canvas)`

I believe that this solves most of the use case in the most natural way. In the future, we could consider:
* Adding a `without <field>` to records to explicitly prevent specific fields. This would also be useful for muxers/demuxers and it's pretty likely we'll get to it.
* Refine type annotation to allow open type with specific fields, i.e. `source(audio=pcm, video=canvas, 'a)`. I think we should get to this one when we need it.